frontend/src/components/core/EnterNickname.tsx

```diff
@@ -79,7 +79,7 @@ export default function EnterNicknamePage(props: EnterNicknameProps) {
           }
         }}
       />
-      <LargeInputButton
+      <PrimaryInput
```

Comment:
I also talked about this in [this screencast](https://www.loom.com/share/3e8affa5e05f43cdb67c4db248bc4acb) (referenced elsewhere as well) - I think this page looks best when the button matches the input size, and when the button is able to be "disabled" as well, but of course open to a discussion on this.

Reply:
Updated